src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -465,6 +465,12 @@ protected function getErrorMessageForException(Exception $e, array $messages)
         $exceptionType = get_class($e);
         $exceptionCode = $e->getCode();
 
+        $this->get('prestashop.core.hook.dispatcher')->dispatchWithParameters('actionGetErrorMessagesInController', [
```

Comment:
I'd change it to `actionSetErrorMessagesInController` or `actionAddErrorMessagesInController` as this hook is meant for extending messeges and not getting them, wdyt?

Reply:
We have another idea - again 😅. Pablo is worried by another usecase.
We'll give it a try tomorrow morning.